docs/Adding-Software-to-Homebrew.md

```diff
@@ -8,46 +8,139 @@ Before you start, please check the open pull requests for [Homebrew/homebrew-cor
 
 Next, you will want to go through the [Acceptable Formulae](Acceptable-Formulae.md) or [Acceptable Casks](Acceptable-Casks.md) documentation to determine if the software is an appropriate addition to Homebrew. If you are creating a formula for an alternative version of software already in Homebrew (e.g. a major/minor version that differs significantly from the existing version), be sure to read the [Versions](Versions.md) documentation to understand versioned formulae requirements.
 
-If everything checks out, you're ready to get started on a new formula!
+If everything checks out, you're ready to get started on a new formula / cask!
```

Comment:
```suggestion
If everything checks out, you're ready to get started on a new formula or cask!
```

Reply:
Fixed in 1de9399aa7fb529f1387491ba1359ea71be5f831